numba/ir.py

```diff
@@ -267,14 +268,14 @@ def cast(cls, value, loc):
     def __repr__(self):
         if self.op == 'call':
             args = ', '.join(str(a) for a in self.args)
-            kws = ', '.join('%s=%s' % (k, v) for k, v in self.kws)
+            kws = ', '.join('%s=%s' % (k, v) for k, v in (self._kws.items() if config.DIFF_IR == 0 else sorted(self._kws.items())))
```

Comment:
I think the == 0 is needed.  If NUMBA_DIFF_IR is 0 then they don't care about ordering and so _kws.items() is used which isn't in deterministic order.  If DIFF_IR is non-zero then used the sorted version.  I presume that you want the predominant case to occur before the "if" and the rare case in the else clause.

Shortened the line.